pkg/git/main.go

```diff
@@ -64,8 +70,14 @@ func IsGitPlan(plan string) bool {
 }
 
 // GetGitPlan will pull git repository and return its path
-func GetGitPlan(plan string, localShuttleDirectoryPath string, uii ui.UI, skipGitPlanPulling bool) string {
+func GetGitPlan(plan string, localShuttleDirectoryPath string, uii ui.UI, skipGitPlanPulling bool, planArgument string) string {
 	parsedGitPlan := parseGitPlan(plan)
+
+	if strings.Index(planArgument, "#") == 0 {
```

Comment:
One could use `strings.HasPrefix()` again if one likes 👀 

```suggestion
	if strings.HasPrefix(planArgument, "#") {
```